src/Components/SystemDetail/RecommendationsTable.scss

```diff
@@ -2,15 +2,35 @@
   tr {
     --pf-c-table--cell--FontSize: var(--pf-global--FontSize--md);
   }
-  .newline {
-    white-space: pre-line;
-  }
-  .tab {
-    white-space: pre-wrap;
-  }
   .recs-table-empty {
     .pf-c-empty-state__icon {
       fill: var(--pf-global--success-color--100);
     }
   }
 }
+.newline {
+  white-space: pre-line;
+}
+.tab {
+  white-space: pre-wrap;
+}
+Th {
+  font-weight: normal;
+}
+strong {
```

Comment:
@r14chandra how about creating a class and adding a <span> with the class name. The reason being, at the moment it is working as expected but in case anyone will add anything with <strong> in this page this padding style will get applied and one will have to check why it is happening. its a small thing but let me know what you think!

Reply:
Yes that's right. I'll do the fix.